package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "npx hardhat clean && npx cpx './packages/tasks/templates/**/*' ./dist/tasks/templates && npx hardhat compile --force && tsc && npx cpx './typechain-types/**/*' ./dist/typechain-types",
+    "build": "npx hardhat clean && npx cpx './packages/tasks/templates/**/*' ./dist/tasks/templates && npx hardhat compile --force && tsc && npx cpx 'dist/packages/*/**' dist && rimraf dist/packages && npx cpx './typechain-types/**/*' ./dist/typechain-types",
```

Comment:
I suggest this to be splitted into smaller scripts for readability, i.e.:
```
"copy-templates": "npx cpx './packages/tasks/templates/**/*' ./dist/tasks/templates",
"copy-packages": "npx cpx 'dist/packages/*/**' dist",
"copy-types": "npx cpx './typechain-types/**/*' ./dist/typechain-types",
"build": "npx hardhat clean && npm run copy-templates && npx hardhat compile --force && tsc && npm run copy-packages && rimraf dist/packages && npm run copy-types"
```

Reply:
cf99a19